### PR TITLE
RSDK-4425 Power sensor wrappers

### PR DESCRIPTION
**Pull request by benjirewis**
https://viam.atlassian.net/browse/RSDK-4425

Adds wrappers/tests for the power sensor.

(cc @acmorrow)

### CODE REVIEW

**Comment by benjirewis** [comment]
Will leave this is in draft until #174 is merged and we decide on a clearer path forward for server and client wrappers for basic gRPC handler methods.

File: src/viam/sdk/components/power_sensor/client.hpp
Comment (benjirewis) [inline src/viam/sdk/components/power_sensor/client.hpp:40]: Not sure if we wanted to merge #180 before this; happy to remove this and use the `ClientHelper` if so. I'm thinking of using https://viam.atlassian.net/browse/RSDK-3556 to propagate `ClientHelper` and `ServiceHelper` everywhere else.

Comment (acmorrow) [reply src/viam/sdk/components/power_sensor/client.hpp:40]: I don't think it matters which way they merge. If you merge this before #180, then I will convert the `stub_wrappers` added in this review over to `ClientHelper` calls. If it goes the other way, you can take it in this review before merge.

I think RSDK-3556 is a fine ticket to use to track the conversion, and I agree that it should be done with as few fly-by changes as possible. In particular, I'd recommend explicitly calling out any such fly-by changes with preemptive comments on the PR so that reviewers are aware of them. The temptation to skim such a large and mechanical review is high.

**Review by acmorrow — COMMENTED** [review]
I don't have much to add on this review, and I'm happy with any sequencing w.r.t. to the ClientHelpers. No need to wait for an approval from me.

**Review by acmorrow — APPROVED** [review]

**Review by stuqdog — APPROVED** [review]
lgtm!

File: src/viam/sdk/components/power_sensor/client.hpp
Comment (benjirewis) [reply src/viam/sdk/components/power_sensor/client.hpp:40]: Since I'm OOO tomorrow I'll merge this PR before #180.

**Pull request merged**
